src/Dialogs/Imap/ImapDialog.vala

```diff
@@ -161,6 +176,17 @@ public class OnlineAccounts.ImapDialog : Hdy.Window {
         smtp_encryption_combobox.append ("starttls-on-standard-port", "STARTTLS");
         smtp_encryption_combobox.active = 2;
 
+        var smtp_auth_label = new Gtk.Label (_("Authentication:")) {
+            xalign = 1
+        };
+
+        smtp_auth_combobox = new Gtk.ComboBoxText () {
+            hexpand = true
+        };
+        smtp_auth_combobox.append ("PLAIN", "PLAIN");
+        smtp_auth_combobox.append ("LOGIN", "LOGIN");
+        smtp_auth_combobox.active = 1;
```

Comment:
Is it intentional that the auth method here is different from the default for imap?